tools/autograd/gen_autograd.py

```diff
@@ -68,6 +66,21 @@
 # returns a view of its inputs
 RETURNS_VIEWS_OF_INPUT = set(VIEW_FUNCTIONS.keys()).union({'chunk', 'narrow'})
 
+# note: some VIEW_FUNCTIONS are returning multiple views. This means that the engine
+# has to forbid inplace operations on their reults. If such function is only doing views
```

Comment:
nit: results

Reply:
This comment says "some view functions return multiple views; therefore, we must forbid inplace operations on results." But WHY do I have to forbid inplace operations on the results? The comment doesn't say. Adding to the reader's confusion, you then contradict this statement immediately below by saying we can create each output as output of a separate node. And the second condition "modification of the output will change the memory usage of the backward" isn't actionable: this is more of a gotcha for users to worry about, not something that would affect whether or not a developer would put the backward in this list.

Reply:
Updated to make it clearer.